analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -574,6 +574,7 @@ open class Npm(
         rawModuleInfoCache[cacheKey]?.let { return it }
 
         val packageJsonFile = moduleDir.resolve("package.json")
+        log.debug { "Parsing module info from '${packageJsonFile.absolutePath}'." }
```

Comment:
I believe it'd be fine to squash this into the previous commit, esp. if the first commit is changed to simply drop the logs.